common/src/attestations/signers.rs

```diff
@@ -38,48 +39,56 @@ pub fn attestation_signers(
         let _p1 = indexer_allocations.pipe(move |allocatons| {
             let _ = allocations_tx.send(allocatons);
         });
-
+        modify_sigers(
+            Arc::new(indexer_mnemonic.clone()),
+            chain_id,
+            attestation_signers_map,
+            allocations_rx.clone(),
+            dispute_manager_rx.clone(),
+        )
+        .await;
```

Comment:
My suggestion is to move this to before line 31, and then you can use it as input for `watch::channel`, so you don't initialize with an empty hashmap

Reply:
ok

Reply:
But modify_signers is an async fn as of now, I'll try to find a solution

Reply:
Is it possible to turn `attestation_signers` into async?

Reply:
sure, I'll try

Reply:
I have email notifications 😆

Reply:
Copy that✌️,